### PR TITLE
New changes

### PR DESCRIPTION
**Pull request by Amit071296**

### CODE REVIEW

**Review by Sahilnagra — APPROVED** [review]

File: src/Pages/Services/IotDevelopment.jsx [hunk 1]
```diff
@@ -17,50 +66,321 @@ const IotDevelopment = () => {
...
-  // button: "Get Free Demo", //uncomment to add button to banner
-  // link: "link" //uncomment to add link to button
 };
 const potentialData = {
-    firstHeading: "360 Degree End-to-End",
-    secondHeading: "IoT & IIoT Services",
-    description: "OnGraph delivers Product Design and Development across industry verticals. Our experts assist you from concept to production that enables you to focus on product strategy and management. Backed by skilled and proficient experts, we make use of the most advanced technology and tools to design and develop IoT products that meet your business model as well as focus on end consumer needs.",
-    cardData: [
+  firstHeading: "360 Degree End-to-End",
+  secondHeading: "IoT & IIoT Services",
```
Comment (Pooja-Sa3) [inline src/Pages/Services/IotDevelopment.jsx:97]: correct Iot spelling

File: src/Pages/Services/IotDevelopment.jsx [hunk 2]
```diff
@@ -17,50 +66,321 @@ const IotDevelopment = () => {
...
+  description:
+    "OnGraph delivers Product Design and Development across industry verticals. Our experts assist you from concept to production that enables you to focus on product strategy and management. Backed by skilled and proficient experts, we make use of the most advanced technology and tools to design and develop IoT products that meet your business model as well as focus on end consumer needs.",
+  cardData: [
```
Comment (Pooja-Sa3) [inline src/Pages/Services/IotDevelopment.jsx:100]: Haspper delivers....

Please don't put OnGraph on descriptions like that

File: src/Pages/Solutions/BlockChain.jsx [hunk 3]
```diff
@@ -103,8 +102,6 @@ const bannerTextData = {
   rightImage: blockchainImg,
   bottomHeading:
   `Dive into the transformative world of secure, reliable, and efficient solutions with our cutting-edge Blockchain technology development. Connect with our top developers for Custom Blockchain Development Services.`,
-     // button: "Get Free Demo", //uncomment to add button to banner
-  // link: "link" //uncomment to add link to button
 };
```
Comment (Pooja-Sa3) [inline src/Pages/Solutions/BlockChain.jsx:105]: remove commented code

File: src/Pages/Services/IotDevelopment.jsx [hunk 4]
```diff
@@ -17,50 +66,321 @@ const IotDevelopment = () => {
...
+    },
+    {
+      icon: multilingualChatbots,
+      number: "1000+",
+      label: "Clients Worldwide",
+    },
+    {
+      icon: multilingualChatbots,
+      number: "98%",
+      label: "Success Rate",
```
Comment (Pooja-Sa3) [inline src/Pages/Services/IotDevelopment.jsx:171]: why are we using same icon for all cards?

**Review by Pooja-Sa3 — CHANGES_REQUESTED** [review]
few small comments need to check before merging, Thanks in advance @Amit071296 !

File: src/Pages/Solutions/BlockChain.jsx (hunk 3 above)
Comment (Amit071296) [reply src/Pages/Solutions/BlockChain.jsx:105]: done

File: src/Pages/Services/IotDevelopment.jsx (hunk 2 above)
Comment (Amit071296) [reply src/Pages/Services/IotDevelopment.jsx:100]: done

File: src/Pages/Services/IotDevelopment.jsx (hunk 1 above)
Comment (Amit071296) [reply src/Pages/Services/IotDevelopment.jsx:97]: done

**Pull request merged**